tasks/phylogenetic_inference/task_centroid.wdl

```diff
@@ -0,0 +1,46 @@
+version 1.0
+
+task centroid {
+  input {
+    Array[File] assembly_fasta
+    Int disk_size = 50
+    Int cpu = 1
+    Int memory = 4
+    String docker = "quay.io/theiagen/centroid:0.1.0"
+  }
+  command <<<
+    centroid.py --version | tee CENTROID_VER
+    
+    mkdir INPUT_DIR
+    # copy all asms to a single directory
+    cp -v ~{sep=' ' assembly_fasta} INPUT_DIR
+    
+    # centroid.py expects a positional argument with a path to a directory of FASTA files
+    centroid.py INPUT_DIR/
+    
+    # set bash variable which ONLY has the filename, e.g. "SAMN19774644_contigs.fasta"
+    CENTROID_FASTA_FILENAME=$(cat centroid_out.txt)
+
+    # rename the centroid genome so it can be accessed outside this task
+    mv -v INPUT_DIR/"$CENTROID_FASTA_FILENAME" centroid.fasta
+
+    # capture samplename of centroid genome for use downstream
+    cut -d '_' -f 1 centroid_out.txt | tee CENTROID_GENOME_SAMPLENAME
```

Comment:
potential future bug: this will not work in the case where the sample name has an underscore, like `sample_01` as the `centroid_genome_samplename` will just be `sample`. We might want to do something where we keep everything except the ending `_contigs.fasta`, assuming they ran the sample through one of our workflows which gives the file that suffix.

Reply:
perhaps cut on `.` instead of underscore